### ISSUE TITLE
get downloaded size without start download file

### ISSUE THREAD

**Issue by MahdiPishguy** [MindorksOpenSource/PRDownloader#20]
is any solution to get downloaded size without start downloading file such percent and download size?

**Comment by amitshekhariitbhu** [comment]
Yes, we can add this API in the new version very soon.

**Comment by cerberusv2px** [comment]
Is this available yet?

**Comment by Zaraki596** [comment]
> Is this available yet? Been almost a year for this thread.

Yes, There is For file size you can get it on OnProgressListener under onProgress method
You just have to call progess.totalBytes
and for File downloaded you can get as progress.currentBytes

**Comment by sindhu-nir** [comment]
Not getting file size by calling progress.totalBytes inside OnProgressListener . It is returning -1 .
What to do ?

**Comment by kmgicrockon** [comment]
> Not getting file size by calling progress.totalBytes inside OnProgressListener . It is returning -1 .
> What to do ?

I also facing the same issue.

**Comment by kartik1225** [comment]
The same issue facing.

**Comment by kayesn786** [comment]
Same issue here. it is returning -1 for files greater than 1 MB

**Comment by bilalhussainn** [comment]
Same Issue. getting totalBytes as -1.
